public/scss/_base.scss

```diff
@@ -15,6 +15,32 @@ iframe,
   &:not([disabled]):focus { @include focus(); }
 }
 
+#skip-link-container {
+  width: 100%;
+  position: absolute;
+  z-index: 5;
+  text-align: center;
+  top: 10px;
```

Comment:
We could use our generic spacing units for this, but I think just setting an hard-coded value is better (like you've done).

The skip link sits more or less outside of our spacing system as such, so this is a good way to go.